### DISCUSSION TITLE
"pip freeze" outputs

### DISCUSSION THREAD

**Discussion by markkuleinio** [netbox-community/netbox#8647] [General]
I don't know if anyone (including me!) would _really_ need this in the future, but here we go anyway:

https://github.com/markkuleinio/netbox-freezes

> Repository for "pip freeze" outputs for original NetBox installations

Started today, with the release of NetBox 3.1.8.

**Comment by candlerb** [comment]
Yes! Reproducible builds FTW!

**Reply by kkthxbye-code** [reply]
Maybe a feature request to investigate the possibility of switching to pipenv or poetry (which both support lock files) could be an idea?
